packages/languages/id/src/wikipedia.json

```diff
@@ -0,0 +1 @@
"wailers","moonstone","assizes","micrometers","hornchurch","truncation","annapurna","egyptologists","rheumatic","promiscuity","satiric","fleche","caloptilia","anisotropy","quaternions","gruppo","viscounts","awardees","aftershocks","sigint","concordance","oblasts","gaumont","stent","commissars","kesteven","hydroxy","vijayanagar","belorussian","fabricius","watermark","tearfully","mamet","leukaemia","sorkh","milepost","tattooing","vosta","abbasids","uncompleted","hedong","woodwinds","extinguishing","malus","multiplexes","francoist","pathet","responsa","bassists","'most","postsecondary","ossory","grampian","saakashvili","alito","strasberg","impressionistic","volador","gelatinous","vignette","underwing","campanian","abbasabad","albertville","hopefuls","nieuwe","taxiways","reconvened","recumbent","pathologists","unionized","faversham","asymptotically","romulo","culling","donja","constricted","annesley","duomo","enschede","lovech","sharpshooter","lansky","dhamma","papillae","alanine","mowat","delius","wrest","mcluhan","podkarpackie","imitators","bilaspur","stunting","pommel","casemate","handicaps","nagas","testaments","hemings","necessitate","rearward","locative","cilla","klitschko","lindau","merion","consequential","antic","soong","copula","berthing","chevrons","rostral","sympathizer","budokan","ranulf","beria","stilt","replying","conflated","alcibiades","painstaking","yamanashi","calif.","arvid","ctesiphon","xizong","rajas","caxton","downbeat","resurfacing","rudders","miscegenation","deathmatch","foregoing","arthropod","attestation","karts","reapportionment","harnessing","eastlake","schola","dosing","postcolonial","imtiaz","formula_55","insulators","gunung","accumulations","pampas","llewelyn","bahnhof","cytosol","grosjean","teaneck","briarcliff","arsenio","canara","elaborating","passchendaele","searchlights","holywell","mohandas","preventable","gehry","mestizos","ustinov","cliched","'national","heidfeld","tertullian","jihadist","tourer","miletus","semicircle","outclassed","bouillon","cardinalate","clarifies","dakshina","bilayer","pandyan","unrwa","chandragupta","formula_56","portola","sukumaran","lactation","islamia","heikki","couplers","misappropriation","catshark","montt","ploughs","carib","stator","leaderboard","kenrick","dendrites","scape","tillamook","molesworth","mussorgsky","melanesia","restated","troon","glycoside","truckee","headwater","mashup","sectoral","gangwon","docudrama","skirting","psychopathology","dramatised","ostroleka","infestations","thabo","depolarization","wideroe","eisenbahn","thomond","kumaon","upendra","foreland","acronyms","yaqui","retaking","raphaelite","specie","dupage","villars","lucasarts","chloroplast","werribee","balsa","ascribe","havant","flava","khawaja","tyumen","subtract","interrogators","reshaping","buzzcocks","eesti","campanile","potemkin","apertures","snowboarder","registrars","handbooks","boyar","contaminant","depositors","proximate","jeunesse","zagora","pronouncements","mists","nihilism","deified","margraviate
```

Comment:
This looks like the english wikipedia we need the Indonesia one

Reply:
I'm sorry, I forget to change the wikipedia.json to Indonesia. I'll change it ASAP.